views/verified.ejs

```diff
@@ -11,9 +11,9 @@
 </head>
 
 <body>
-    <h1>Your account has been verified! Yay!</h1>
+    <h1>Your account has been verified! Yay! 🎉</h1>
     <a href="./login">
-        <button>Go login?</button>
+        <button>Login?</button>
```

Comment:
this file is never used, i need to remove it.

Reply:
Yea i don't remember ever seeing it

Reply:
I'll remove it lemme push that